TombEngine/Scripting/Internal/TEN/Objects/Moveable/MoveableObject.cpp

```diff
@@ -424,7 +425,29 @@ ScriptReserved_GetSlotHP, & Moveable::GetSlotHP,
 // @tparam Objects.ObjID ObjectID to take animation and stateID from,
 // @tparam int animNumber animation from object
 // @tparam int stateID state from object
-	ScriptReserved_AnimFromObject, &Moveable::AnimFromObject);
+	ScriptReserved_AnimFromObject, &Moveable::AnimFromObject,
+		
+	/// Get a creature moveable's current enemy
+	// @Function Moveable:GetCreatureEnemy
+	ScriptReserved_GetCreatureEnemy, &Moveable::GetCreatureEnemy,
+
+	/// Clears a creature moveable's current enemy
+	// @Function Moveable:ClearCreatureEnemy
+	ScriptReserved_ClearCreatureEnemy, &Moveable::ClearCreatureEnemy,
+
+	/// Set a creature moveable's current enemy
+	// @Function Moveable:SetCreatureEnemy
+	// @tparam Moveable A moveable for the creature to treat as enemy
```

Comment:
Remove the `A` . This is because , the generated documentation generator reads this wrong.